packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -14,6 +15,12 @@ import {
 export const RESIZE_HANDLE_CLASS = 'dx-resize-handle';
 const HORIZONTAL_DIRECTION_CLASS = 'dx-resize-handle-horizontal';
 const VERTICAL_DIRECTION_CLASS = 'dx-resize-handle-vertical';
+const RESIZE_HANDLE_ICON_CLASS = 'dx-resize-handle-icon';
+const RESIZE_HANDLE_COLLAPSE_PREV_BUTTON_CLASS = 'dx-resize-handle-collapse-prev-button';
```

Comment:
These class names should be approved by @vladaskorohodova.

Reply:
I don't exactly understand what do you mean by "previous" and "next" buttons. Can you clarify?

Reply:
@vladaskorohodova there are two splitter panes (before and after this handle), collapse prev should hide a pane before a resize_handle, collapse next should hide a pane after a risize_handle

Reply:
How about
`dx-resize-handle-collapse-prev-button` -> `dx-resize-handle-collapse-previous-pane-button`
`dx-resize-handle-collapse-next-button` -> `dx-resize-handle-collapse-next-pane-button`